doc/guides/compile.md

```diff
@@ -338,6 +338,47 @@ When debugging mode is enabled
   * Because `-g` flag would be added to `mrbc` runner.
     * You can have better backtrace of mruby scripts with this.
 
+### Additional license terms
+
+The license terms set in the `conf.terms` property are printed as part of
+the `<build-dir>/LICENSE.yml` file.
+This initial value is an empty array (`[]`). If you want to add it, do the
+following:
+
+```ruby
+conf.terms << "path/to/ADDITIONAL-LICENSE"
+```
+
+The file path can be specified as an absolute path as well as a relative
+path based on the build directory.
+Also, you can give not only the file path but also a hash that combines
```

Comment:
```suggestion
Optionally, you can pass a hash that combines the name and the content of the terms.
```

Reply:
The `terms` property is now just a string, so it's no longer explained.